orchestrator/api/verticals.py

```
@@ -0,0 +1,248 @@
+    public_id: str
+    name: str
+    api_key: str = Field(..., description="Public widget API key — shown once.")
+    agents_installed: int
+    is_new: bool
+
+
+@router.post("/{vertical}/provision", response_model=VerticalProvisionResponse)
+async def provision_vertical_workspace(
+    vertical: str,
+    request: VerticalProvisionRequest,
+    db: Session = Depends(get_db),
```

Ruff (0.15.20)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)